pkg/vm.go

```diff
@@ -0,0 +1,113 @@
+package pkg
+
+import (
+	"fmt"
+	"net"
+	"os"
+)
+
+//go:generate zbusc -module vmd -version 0.0.1 -name manager -package stubs github.com/threefoldtech/zos/pkg+VMModule stubs/vmd_stub.go
+
+// VMNetworkInfo structure
+type VMNetworkInfo struct {
+	// Namespace name
+	Namespace string
+	// Tap device name
+	Tap string
+	// Mac address of the device
+	MAC net.HardwareAddr
+	// Address of the device in the form of cidr
+	AddressCIDR string
```

Comment:
why not using a proper type for it, `net.IPNet` ?

Reply:
Hmm, i actually started with using both net.IPNet and net.IP types first and then switched to strings, don't quiet remember why, but at this moment I assume it makes client code easier. (provisiond) ? should I switch back to `net.` types.

Reply:
I prefer to use proper types. Anything can be stored into string, this just force more conversion along the way I think.